app/src/main/java/com/example/android/weatherapp/adapter/viewholder/CurrentWeatherViewHolder.kt

```diff
@@ -7,21 +7,33 @@ import androidx.recyclerview.widget.RecyclerView
 import com.example.android.weatherapp.R
 import com.example.android.weatherapp.models.CurrentWeatherModel
 import kotlinx.android.synthetic.main.current_weather_layout.view.*
+import java.text.SimpleDateFormat
+import java.util.*
 
-class CurrentWeatherViewHolder (itemView: View): RecyclerView.ViewHolder(itemView) {
+private const val DATE_FORMAT= "MMMM dd, yyyy"
+
+class CurrentWeatherViewHolder(itemView: View): RecyclerView.ViewHolder(itemView) {
 
     fun bindingViewToData(item: CurrentWeatherModel) {
-       itemView.current_weather_type_text.text = item.weather[0].main
+        itemView.current_weather_type_text.text = item.weather[0].main
         itemView.current_temp_text.text = item.main.temp.toString()
-        /*itemView.current_location_text.text = item.location
-        itemView.current_date_text.text = item.date*/
+        itemView.current_location_text.text = item.name
+        itemView.current_date_text.text = getCurrentDate()
+    }
+
+    private fun getCurrentDate(): String {
```

Comment:
This method can be moved to a separate helper class. It makes sure that we don't bloat our view holder with extra code.

Reply:
Create helper class in pull request#4